lib/tty-error.js

```diff
@@ -8,6 +8,8 @@ var cwd = process.cwd()
 module.exports = ttyError
 
 function ttyError (src, sub, err) {
+  if (!err.loc) return err
```

Comment:
I woul do `if (!err.filename || !err.loc) return err`. (See screenshot below for one of the errors I was running into...)

<img width="1230" alt="screen shot 2017-10-01 at 10 21 10 am" src="https://user-images.githubusercontent.com/970121/31055552-a08a43ec-a692-11e7-9c97-6d1a6e3f740b.png">

Reply:
otherwise LGTM